docs/create-sections.html

```diff
@@ -3,5 +3,28 @@
 title: Create Sections
 ---
 
-<h1 class="mt-5">Create Sections Documentation</h1>
-<p>Follow documentation for this Feature</p>
\ No newline at end of file
+<h1 class="mt-5">Create Sections</h1>
+<p>
+  Some courses benefit from additional sections to organize students/participants for a variety of reasons. 
+  Course Manager makes new section creation quick and easy by uploading a simple CSV file.
+</p>
+<p>
+  <b>Tip: </b>Authorized users can add sections on the fly when adding U-M or non-U-M users with this tool. 
+  See the help documentation for these features for more information.
+</p>
+<h2 class="h5">Who Can Create Sections</h2>
+<p>Section management activities in this tool are limited to administrator roles:</p>
```

Comment:
I think this needs to be "creation", since teachers can use "Merge Sections" which I would consider a section management activity.
```suggestion
<p>Section creation activities in this tool are limited to administrator roles:</p>
```

Reply:
I will stick with what I have before.

Reply:
This just feels inaccurate though.

Reply:
The is the section creation tool, there for "creation."

Reply:
@melindakraft if you like any SAm's Suggesting you can commit it now since That is is quickest thing to do.